Libraries/LibWeb/CSS/StyleValues/CSSRGB.cpp

```diff
@@ -71,6 +71,8 @@ bool CSSRGB::equals(CSSStyleValue const& other) const
 String CSSRGB::to_string() const
 {
     // FIXME: Do this properly, taking unresolved calculated values into account.
+    if (m_properties.name.has_value())
+        return m_properties.name.value().to_string();
```

Comment:
[Named colors](https://drafts.csswg.org/css-color-4/#named-color) only retain their name when the [declared value](https://drafts.csswg.org/css-cascade-5/#declared-value) is needed. When the [used value](https://drafts.csswg.org/css-cascade-5/#used-value) or [computed value](https://drafts.csswg.org/css-cascade-5/#computed-value) is required, we still need to return the sRGB value, as before.

See here for more information: https://drafts.csswg.org/css-color-4/#serializing-sRGB-values.

For example [`getComputedStyle()`](https://drafts.csswg.org/cssom/#dom-window-getcomputedstyle) requires that the used value be returned for color properties: https://drafts.csswg.org/cssom/#resolved-value.

Here are a couple of WPT tests that use `getComputedStyle()` and regress with the current change: 
* http://wpt.live/css/cssom/getComputedStyle-display-none-001.html
* http://wpt.live/css/cssom/getComputedStyle-display-none-003.html

Reply:
The regressions you listed have now been fixed, by removing color names in the style computer.